news/2019-07-07-beatmap-spotlights-spring-2019.md

```diff
@@ -0,0 +1,302 @@
+
+Those unprepared for the challenge may find a very engaging experience as well in the insane difficulty, although skilled players would not wish to sleep on this one too. With more emphasis on greater slider movements, the difficulties compliment one another with their slight varied approaches to the song, while both capturing its integrity faithfully. A truly refreshing mapset overall that sits distinguished among the many maps in the ranked section!
+
+_written by [faygo](https://osu.ppy.sh/users/5199686)
+
+[![](/wiki/shared/news/2019-07-07-beatmap-spotlights-spring-2019/osu/uh.jpg)](https://osu.ppy.sh/beatmapsets/948199)
+
+[sammish](https://osu.ppy.sh/users/4415076) manages to make 17 minutes feel like 5 in his map of Tennyson's "Uh Oh! EP". The hard work he puts into expressing each song's personality shows- and obviously pays off. The map feels smooth to play all the way through and aligns very closely to Tennyson's unique, funky style of music. This is a phenomenal debut of both sammish and Tennyson in the ranked section and we hope to see more from both in the future!
+
+_written by [squirrelpascals](https://osu.ppy.sh/users/6151332)
+
+[![](/wiki/shared/news/2019-07-07-beatmap-spotlights-spring-2019/osu/meng.jpg)](https://osu.ppy.sh/beatmapsets/927682)
+
+[AIR](https://osu.ppy.sh/users/2070688) displays their usual prowess in Yin Lin's "Lu Gu Xun Meng", a traditional-sounding Chinese ballad. They skillfully craft dynamic jump patterns with expert use of Slider Velocity changes, repeating slider shapes, and varied geometric arrangements. 
+
+As always, the map has a lot of contrast, with every separate section of the map feeling uniquely matched closely to the features of the song. The hitsounding, also done by AIR, follows the same idea, adding to the experience. If the top difficulty's a bit too jumpy, be sure to check out the Insane and Hard, which maintain the same concepts as the top difficulty at a more manageable level.
+
+_written by [Nao Tomori](https://osu.ppy.sh/users/5364763)
+
+[![](/wiki/shared/news/2019-07-07-beatmap-spotlights-spring-2019/osu/kyoumen.jpg)](https://osu.ppy.sh/beatmapsets/731108)
+
+_written by [Nao Tomori](https://osu.ppy.sh/users/5364763)
+
+[![](/wiki/shared/news/2019-07-07-beatmap-spotlights-spring-2019/osu/focus.jpg)](https://osu.ppy.sh/beatmapsets/925329)
+
+Capturing the essence of a song can be considered a herculean feat for a mapper. This map gets the job done. [Vell](https://osu.ppy.sh/users/3414541) once again takes off the breaks and delivers another masterpiece of contemporary mapping with sukekiyo's song "focus". In tune with the music along [Darky1](https://osu.ppy.sh/users/3372928)'s storyboard the map follows the baroque-an motif of vanitas through which it blooms into a thundering roar of raw emotion to then just as quickly fade away into nothingness. All the while showcasing pristine mapping along with the natural melodic texture
```

Comment:
```suggestion
Capturing the essence of a song can be considered a herculean feat for a mapper. This map gets the job done. [Vell](https://osu.ppy.sh/users/3414541) once again takes off the brakes and delivers another masterpiece of contemporary mapping with sukekiyo's song "focus". In tune with the music along [Darky1](https://osu.ppy.sh/users/3372928)'s storyboard, the map follows the baroque-an motif of vanitas through which it blooms into a thundering roar of raw emotion to then just as quickly fade away into nothingness. All the while, it showcases pristine mapping along with the natural melodic texture.
```

Reply:
Capturing the essence of a song can be considered a herculean feat for a mapper. This map gets the job done. [Vell](https://osu.ppy.sh/users/3414541) once again takes off the breaks and delivers another masterpiece of contemporary mapping with sukekiyo's song "focus". In tune with the music along [Darky1](https://osu.ppy.sh/users/3372928)'s storyboard the map follows the baroquean motif of vanitas through which it blooms into a thundering roar of raw emotion to then just as quickly fade away into nothingness. All the while showcasing pristine mapping along with the natural melodic texture.

Reply:
```suggestion
Capturing the essence of a song can be considered a herculean feat for a mapper. This map gets the job done. [Vell](https://osu.ppy.sh/users/3414541) once again takes off the breaks and delivers another masterpiece of contemporary mapping with sukekiyo's song "focus". In tune with the music along [Darky1](https://osu.ppy.sh/users/3372928)'s storyboard the map follows the baroque-an motif of vanitas through which it blooms into a thundering roar of raw emotion to then just as quickly fade away into nothingness. All the while showcasing pristine mapping along with the natural melodic texture
```

Reply:
```suggestion
Capturing the essence of a song can be considered a herculean feat for a mapper. This map gets the job done. [Vell](https://osu.ppy.sh/users/3414541) once again takes off the breaks and delivers another masterpiece of contemporary mapping with sukekiyo's song "focus". In tune with the music along [Darky1](https://osu.ppy.sh/users/3372928)'s storyboard the map follows the baroquean motif of vanitas through which it blooms into a thundering roar of raw emotion to then just as quickly fade away into nothingness. All the while showcasing pristine mapping along with the natural melodic texture.
```

Reply:
```suggestion
Capturing the essence of a song can be considered a herculean feat for a mapper. This map gets the job done. [Vell](https://osu.ppy.sh/users/3414541) once again takes off the brakes and delivers another masterpiece of contemporary mapping with sukekiyo's song "focus". In tune with the music along [Darky1](https://osu.ppy.sh/users/3372928)'s storyboard, the map follows the baroquian motif of vanitas through which it blooms into a thundering roar of raw emotion to then just as quickly fade away into nothingness. All the while, it showcases pristine mapping along with the natural melodic texture.
```

Reply:
```suggestion
Capturing the essence of a song can be considered a herculean feat for a mapper. This map gets the job done. [Vell](https://osu.ppy.sh/users/3414541) once again takes off the brakes and delivers another masterpiece of contemporary mapping with sukekiyo's song "focus". In tune with the music along [Darky1](https://osu.ppy.sh/users/3372928)'s storyboard, the map follows the baroquian motif of vanitas through which it blooms into a thundering roar of raw emotion to then just as quickly fades away into nothingness. All the while, it showcases pristine mapping along with the natural melodic texture.
```
fade -> fades